Doc/howto/unicode.rst

```diff
@@ -156,7 +157,7 @@ UTF-8 has several convenient properties:
 References
 ----------
 
-The `Unicode Consortium site <https://www.unicode.org>`_ has character charts, a
```

Comment:
What was wrong with this URL?

Reply:
This may be more nitpicky by me and could probably be removed from this PR. My reasoning is as follows:

This part of the HOWTO says:
![image](https://user-images.githubusercontent.com/32741226/78499485-a4dd9080-7759-11ea-8f20-267b1722decb.png)

To me
> character charts, a glossary, and PDF versions of the Unicode specification

suggests technical detail. The page at https://www.unicode.org redirects one to https://home.unicode.org/ which is more of a "_**marketing**_" than a "_**technical**_" page. Eventually after clicking a bit one can get to the technical page (scroll down and click on the "**_Unicode Technical Site_**" link). The technical link, https://www.unicode.org/main.html, is faster to the **Character Charts** (the page calls them code charts), **Glossary** and **Unicode Specifications** (at least in HTML. To get to the PDF version is 2 clicks from either page.)

Reply:
Let's not change this here.